src/BlockID.cpp

```diff
@@ -222,6 +236,11 @@ bool StringToItem(const AString & a_ItemTypeString, cItem & a_Item)
 	{
 		ItemName = ItemName.substr(10);
 	}
+
+	if (gsBlockIDMap.m_bHasRunInit != true)
```

Comment:
You can just use `if (!gsBlockIDMap.m_bHasRunInit)`. It should be faster as well.